examples/correlated_ou/training.jl

```diff
@@ -81,30 +71,19 @@ function run_training(params; FT=Float32, logger=nothing)
     # set up model and optimizers
     checkpoint_path = joinpath(savedir, "checkpoint.bson")
     loss_file = joinpath(savedir, "losses.txt")
-
+    
     if isfile(checkpoint_path) && isfile(loss_file)
         BSON.@load checkpoint_path model model_smooth opt opt_smooth
         model = device(model)
         model_smooth = device(model_smooth)
         loss_data = DelimitedFiles.readdlm(loss_file, ',', skipstart = 1)
-        start_epoch = loss_data[end,1]+1
+        start_epoch = loss_data[end, 1] + 1
     else
-        net = NoiseConditionalScoreNetwork(;
-                                           noised_channels = inchannels,
-                                           shift_input = shift_input,
-                                           shift_output = shift_output,
-                                           mean_bypass = mean_bypass,
-                                           scale_mean_bypass = scale_mean_bypass,
-                                           gnorm = gnorm,
-                                           proj_kernelsize = proj_kernelsize,
-                                           outer_kernelsize = outer_kernelsize,
-                                           middle_kernelsize = middle_kernelsize,
-                                           inner_kernelsize = inner_kernelsize
-                                           )
+        net = NoiseConditionalScoreNetwork3D(; noised_channels = inchannels, nspatial=3)
```

Comment:
is nspatial = 3 and "NoiseConditionalScoreNetwork3D" redundant? does NoiseConditionalScoreNetwork3D support nspatial = 2?

Reply:
I think in the future, we will just have one, so it is redundant now, but later we can just rename and be done if tests pass.